src/pgduckdb_ddl.cpp

```diff
@@ -101,6 +96,66 @@ DuckdbHandleDDL(Node *parsetree) {
 	}
 }
 
+static void
+DuckdbUtilityHook_Cpp(PlannedStmt *pstmt, const char *query_string, bool read_only_tree, ProcessUtilityContext context,
+                      ParamListInfo params, struct QueryEnvironment *query_env, DestReceiver *dest,
+                      QueryCompletion *qc) {
+	Node *parsetree = pstmt->utilityStmt;
+	if (pgduckdb::IsExtensionRegistered() && IsA(parsetree, CopyStmt)) {
```

Comment:
Should we just test for `pgduckdb::IsExtensionRegistered()` at the top of the function once and for all (and return early after calling prev hook)? Is there any case where we might want to do something when this is false?